### ISSUE TITLE
Possible scroll issue in latest version of Chrome ( v98 )

### ISSUE THREAD

**Issue by lluia** [facebook/docusaurus#6754]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#reporting-new-issues).

### Prerequisites

- [X] I'm using the latest version of Docusaurus.
- [X] I have tried the `npm run clear` or `yarn clear` command.
- [X] I have tried `rm -rf node_modules yarn.lock package-lock.json` and re-installing packages.
- [X] I have tried creating a repro with https://new.docusaurus.io.
- [X] I have read the console error message carefully (if applicable).

### Description

Hi! Thanks for the excellent work with **Docusaurus 🦖**, we use it in [**NextAuth.js**](next-auth.js.org/) for our documentation, serving us well 💚

We just had the following issue reported by some users:
https://github.com/nextauthjs/next-auth/issues/4054

Scroll gets hijacked in the latest version of Chrome ( `98.0.4758.102 ` ). The strange thing is that some of our team members have reproduced it while others have not. Here's another reproduction from one of our team members:

https://user-images.githubusercontent.com/5938217/155589220-7cd2dcfb-9b52-4e21-808a-91ca5c1b9b77.mp4

I'm opening this as a discussion as I'm not entirely sure is a Docusaurus issue or a Chrome issue or something caused by a plugin in Chrome 🤔; we don't have clear steps to reproduce it. 

### Steps to reproduce

1. Open **Chrome**
1. Visit https://next-auth.js.org/getting-started/client ( we're using the latest Docusaurus version, `2.0.0-beta15` )
1. Scroll up and down the page

### Expected behavior

Scroll is smooth 🍉

### Actual behavior

Some users are unable to scroll in the latest version of Chrome, or have notable difficulties doing so

### Your environment

- Public source code → https://github.com/nextauthjs/next-auth/tree/main/docs
- Public site URL →  https://next-auth.js.org/getting-started/client 
- Docusaurus version used → `2.0.0-beta15`
- Environment name and version →  Chrome 98
- Operating system and version → MacOS & Linux

### Self-service

- [X] I'd be willing to fix this bug myself.

**Comment by duanwilliam** [comment]
looks the same as #6620 . breaking pr is being temporarily reverted for next release in #6748 , draft/discussion regarding fixes are in #6666

**Comment by Josh-Cena** [comment]
Yep, thanks for that explanation @duanwilliam

**Comment by Josh-Cena** [comment]
TL;DR: turn off the "experimental web platform features" flag

**Comment by balazsorban44** [comment]
Thabks, could you explain what that means? 😵 Where do we turn this off? I don't see anything like this in our config files.

**Comment by Josh-Cena** [comment]
> Where do we turn this off? I don't see anything like this in our config files.

That's a Chrome flag. Sorry, wrong issue that @duanwilliam linked, should be #6587
